Builds in epel8 resolved

majamassarini opened the pull request.

lbarcziova reviewed (commented):
why is this another file and not the original `content/issues/2023-11-14-copr-outage-for-epel8.md`?

majamassarini commented:
> why is this another file and not the original `content/issues/2023-11-14-copr-outage-for-epel8.md`?

yesterday I was trying to understand the same, but honestly I didn't come up with an answer

> why is this another file and not the original `content/issues/2023-11-14-copr-outage-for-epel8.md`?

while rebasing I brought inside the PR a new commit with the new file, I don't know how, now I have deleted the new file and modified the old one.
